src/globals/scss/_colors--experimental.scss

```diff
@@ -0,0 +1,34 @@
+$ibm-colors__exclude-css-variables: true !default;
```

Comment:
Do we need this file for the experimental work?

Reply:
I don't think so, no idea what its even doing.